pet-kata-solutions/src/test/java/org/eclipse/collections/petkata/Exercise4Test.java

```diff
@@ -76,37 +71,35 @@ public void streamsToECRefactor1() {
     }
 
     @Test
-    public void streamsToECRefactor2() {
-        // Hint: Try to replace the Map<PetType, Long> with a Bag<PetType>
-        MutableBag<PetType> counts = this.people
+    public void streamsToECRefactor2()
+    {
+        MutableBag<PetType> countsStream = this.people
                 .asUnmodifiable()
                 .flatCollect(Person::getPets)
                 .countBy(Pet::getType);
 
-        Assert.assertEquals(2, counts.occurrencesOf(PetType.CAT));
-        Assert.assertEquals(2, counts.occurrencesOf(PetType.DOG));
-        Assert.assertEquals(2, counts.occurrencesOf(PetType.HAMSTER));
-        Assert.assertEquals(1, counts.occurrencesOf(PetType.SNAKE));
-        Assert.assertEquals(1, counts.occurrencesOf(PetType.TURTLE));
-        Assert.assertEquals(1, counts.occurrencesOf(PetType.BIRD));
+        Assert.assertEquals(2, countsStream.occurrencesOf(PetType.CAT));
+        Assert.assertEquals(2, countsStream.occurrencesOf(PetType.DOG));
+        Assert.assertEquals(2, countsStream.occurrencesOf(PetType.HAMSTER));
+        Assert.assertEquals(1, countsStream.occurrencesOf(PetType.SNAKE));
+        Assert.assertEquals(1, countsStream.occurrencesOf(PetType.TURTLE));
+        Assert.assertEquals(1, countsStream.occurrencesOf(PetType.BIRD));
     }
 
-
     /**
      * The purpose of this test is to determine the top 3 pet types.
      */
     @Test
-    public void streamsToECRefactor3() {
-        // Hint: The result of groupingBy/counting can almost always be replaced by a Bag
-        // Hint: Look for the API on Bag that might return the top 3 pet types
-        MutableList<ObjectIntPair<PetType>> favorites = this.people
+    public void streamsToECRefactor3()
+    {
+        var favoritesStream = this.people
```

Comment:
Please rename this `topThreeList` or `favorites`.

Reply:
Fixed